CONTRIBUTING.md

```diff
@@ -41,14 +41,11 @@ and then attach IDE Debugger to it.
 
 To compile and launch a development instance, run:
 
-    mvn hudson-dev:run
-
-<!--
-This should be `mvn hudson-dev:run` as long as JENKINS-23364 is not resolved.
-For jenkis-dev:run it is currently necessary to add `org.jenkins-ci.tools` as a plugin group
-to the maven settings.xml.
-See also https://github.com/jenkinsci/jenkins/pull/4331#discussion_r341041470
--->
+```sh
+mvn -f war jetty:run
```

Comment:
I get this on the first run:
```
Non-resolvable import POM: Could not find artifact org.jenkins-ci.main:jenkins-bom:pom:2.205-SNAPSHOT in repo.jenkins-ci.org (https://repo.jenkins-ci.org/public/)
```
I assume the above `mvn clean package` will resolve it, but it's not clear that its a required step for running this step

Reply:
Thanks, that worked, documentation above here doesn't look accurate anymore though